### DISCUSSION TITLE
Dns check not working

### DISCUSSION THREAD

**Discussion by Verox2903** [community/community#147118] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

After configuring my custom domain dns check could not complete. Below I post screenshots of my dns configuration.
I have read a ton of diffrent articles about it, but i still can't get it to work. Please help.
![image](https://github.com/user-attachments/assets/cc49e40e-8ce3-4b3a-bc5e-47ed1a3ed092)

**Comment by thelaycon** [comment]
I checked out https://giziu.pl/ and it appears to be linking to the Github page already.

Please note that it takes some time for DNS to propagate. If you make any changes to a DNS setting, wait for some hours for full propagation or reflection.

**Reply by Verox2903** [reply]
This site and connection to github was established propably a month ago so its more than enough time for DNS to propagate.

**Reply by Verox2903** [reply, marked as answer]
I just realized that the dns check completed with no problems. I didn't change anything and it works now.

**Reply by thelaycon** [reply]
Great, enjoy,

**Comment by Pompatcha** [comment]
how long wait for DNS Check in Progress?

Custom domain.

**Reply by Verox2903** [reply]
after you point A, AAAA and CNAME and TXT (for verification on profile), i recommend to have ALIAS poining to www.yourdomain.com not just yourdomain.com and then wait up to 24-48 hours and dns check should take only about a minute to finish.

**Reply by Pompatcha** [reply]
thank I got it!!!!

**Comment by mimnifras** [comment]
***@***.***
